DATA/UNIVERSE/SYSTEMS/Br04/Br04.ini

```diff
@@ -2592,6 +2550,52 @@ interference = 0.7
 visit = 128
 sort = 99.5
 
+; Ambient Zones [Must Come Fourth]
+
+[zone]
+nickname = zone_br04_pop_ambient_n
+pos = -539, 0, -35809
+shape = ELLIPSOID
+size = 130178, 22567, 67545
+comment = Ambient Pop
+sort = 50
+toughness = 19 ;toughness = 5
+density = 3
+repop_time = 45
+max_battle_size = 4
+pop_type = Background
+relief_time = 35
+population_additive = false
+encounter = area_scout, 7, 0.17
+faction = fc_g_grp, 0.29
+faction = fc_m_grp, 0.71
+encounter = area_scout, 9, 0.1
+faction = cl_01_grp, 0.3
+faction = br_n_grp, 0.7
+
+[zone]
+nickname = zone_br04_pop_ambient_s
+pos = 1436, 0, 55592
+shape = ELLIPSOID
+size = 117935, 17630, 54085
+comment = Ambient Pop
+sort = 51
+toughness = 19 ;toughness = 5
+density = 3
+repop_time = 45
+max_battle_size = 4
+pop_type = Background
+relief_time = 35
+population_additive = false
+encounter = area_scout, 7, 0.17
+faction = fc_ou_grp, 0.29
+faction = fc_m_grp, 0.71
+encounter = area_scout, 9, 0.1
+faction = cl_01_grp, 0.3
+faction = br_p_grp, 0.7
+encounter = area_trade_freighter, 11, 0.2
+faction = br_m_grp, 1
+
 ; Exclusion Zones [Must Come Last]

```

Comment:
Just for general info: They do not need to come last. If they have NPC spawns or anything with density, they actually may need to be earlier in the file. But here, since they do nothing to NPCs, it does not matter.